### PR TITLE
moved common logic to GenericDao and GenericService, create web app

### PR DESCRIPTION
**Pull request by mikhail771**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,6 +17,19 @@
...
+        <dependency>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-war-plugin</artifactId>
+            <version>3.3.1</version>
+            <type>maven-plugin</type>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```
Comment (AnaLanda) [inline pom.xml:30]: I think we also need `<scope>provided</scope>` under the version

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    }
+</style>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<div class='rotatable'>
+    <h1 style="text-align: center; color: red;">Hello world!!!</h1>
+</div>
+
```
Comment (AnaLanda) [inline src/main/webapp/WEB-INF/views/index.jsp]: Maybe you should remove the extra newline here

**Review by AnaLanda — COMMENTED** [review]
Looks good overall. Added a couple comments)

File: pom.xml [hunk 3]
```diff
@@ -16,6 +17,19 @@
             https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
     </properties>
+    <dependencies>
+        <dependency>
```
Comment (Sofasmile) [inline pom.xml:21]: should we add to dependency plugin? Does it work without dependency?

File: pom.xml (hunk 1 above)
Comment (Sofasmile) [reply pom.xml:30]: you can leave it as it is

File: pom.xml (hunk 3 above)
Comment (mikhail771) [reply pom.xml:21]: Yes, plugin without dependensy didn't work for me

File: pom.xml (hunk 1 above)
Comment (mikhail771) [reply pom.xml:30]: ok

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 2 above)
Comment (mikhail771) [reply src/main/webapp/WEB-INF/views/index.jsp]: done

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**